ekf/kalman_implem.c

```diff
@@ -146,6 +148,25 @@ static matrix_t *kmn_getCtrl(matrix_t *U)
 }
 
 
+/* Damping acceleration magnitude if its length is below ACC_DAMP_THRESHOLD */
+float kmn_accelDamp(vec_t *accel)
+{
+	float len, damp;
+
+	len = vec_len(accel);
+
+	if (len < ACC_DAMP_THRESHOLD) {
+		len /= ACC_DAMP_THRESHOLD;
+		damp = len * len * len * len;
```

Comment:
Tiny optimization, there's a chance the compiler does this anyway
```suggestion
		len *= len;
		damp = len * len;
```

Reply:
If compiler may do it by itself, I would like to leave this part of code untouched for easy understanding that it is _normalized `len` to the fourth power_.